client/js/sign.js

```diff
@@ -0,0 +1,70 @@
+const inForm = document.getElementById('signin-frm');
+const upForm = document.getElementById('signup-frm');
+
+// POST data and get response to use the result for user check
+async function postData(url = '', data = {}) {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    redirect: 'follow',
+    body: JSON.stringify(data)
+  });
+
+  const status = response.status;
+  const res = await response.json();
+  return { status: status, response: res }
+}
+
+// Check user existence
+inForm.addEventListener('submit', (frm) => {
+  frm.preventDefault();
+  document.querySelector('.alert').style.display = 'none';
+
+  const usrData = {
+    user_name: frm.target.elements[0].value,
+    password: frm.target.elements[1].value
+  }
+
+  postData('/signin', usrData)
+    .then(data => {
+      if (data.status === 200) {
+        return window.location.replace('../html/main.html')
+      } else { 
+        document.querySelector('.user-not-exists').style.display = 'block'
```

Comment:
For possible future use cases, how about refactoring this out to its own smaller function?

Reply:
Will try!